tests/unit/MigrateRunnerTest.php

```diff
@@ -184,6 +182,11 @@ protected function getDatabaseConnection()
         return $connection;
     }
 
+    public static function webroot(): string
+    {
+        return Path::join(Path::canonicalize(dirname(__DIR__, 2)), 'sut');
+    }
+
```

Comment:
Why are we overrideing the webroot from the parent? Seems like it does same thing.

Reply:
The parent of this test is vanilla `TestCase`, so we won't have that there. Unless I'm missing something?